### PR TITLE
Staking hub patch 1 1

### PR DESCRIPTION
**Pull request by staking-hub**
Add light token to chain registry. No chain exists

### CODE REVIEW

File: light/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "$schema":"../assetlist.schema.json",
+    "chain_name":"light",
```
Comment (JeremyParish69) [inline light/assetlist.json:3]: It sounds like this isn't its own chain, but rather a token on lumenx, in which case please add this to the assetlist there instead.

Comment (staking-hub) [reply light/assetlist.json:3]: It is a juno-chain token and has no chain (neither genesis not validators). Should I include it below lumen assetlist?

Comment (JeremyParish69) [reply light/assetlist.json:3]: if it's a juno token then it should go in Juno's assetlist. Why would it go in Lumen's assetlist?

**Comment by staking-hub** [comment]
In fact it was minted using junod.binary, but it is token used for providing liquidity incentives in lumen pools.

LuckyStar

Enviado desde Proton Mail móvil

-------- Mensaje original --------

**Comment by JeremyParish69** [comment]
> In fact it was minted using junod.binary, but it is token used for providing liquidity incentives in lumen pools.

Thanks for explaining. Purpose is irrelevant to this. It should go in Juno's assetlist

**Comment by JeremyParish69** [comment]
Also, the PNG and SVG are too different. I'd suggest just sticking with the PNG

**Comment by staking-hub** [comment]
Done. Removed the .SVG file.
Uploaded the .PNG file into Juno/images
Opened a new PR to list light in juno.

LuckyStar

Enviado desde Proton Mail móvil

-------- Mensaje original --------

**Comment by JeremyParish69** [comment]
LIGHT was added in https://github.com/cosmos/chain-registry/pull/1958

**Pull request closed without merging**